pages/404.js

```diff
@@ -0,0 +1,381 @@
+import * as React from "react";
+import * as Constants from "~/common/constants";
+
+import { css } from "@emotion/react";
+
+import WebsitePrototypeWrapper from "~/components/core/WebsitePrototypeWrapper";
+
+const STYLES_ROOT = css`
+  width: 80%;
```

Comment:
Please use the padding strategy here and adjust the padding as necessary with `@media`